### PR TITLE
Fixes #28391 - fix failing tests

### PR DESCRIPTION
**Pull request by Ron-Lavi**
After one of the latest patternfy-react version update with `react-bootstrap` update, tests started to fail.

opened issue to patternfly-react: https://github.com/patternfly/patternfly-react/issues/3364
opened issue to react-bootstrap: https://github.com/react-bootstrap/react-bootstrap/issues/4872

blocked by https://github.com/theforeman/foreman-js/pull/79

### CODE REVIEW

**Comment by sharvit** [comment]
Thank you @laviro, can you please update the package.json as well to use the new vendor (`3.5.1`)?

**Comment by Ron-Lavi** [comment]
Thanks, updated :)

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^3.3.2",
```
Comment (sharvit) [inline package.json:23]: Need to update all the `@theforeman/*` deps

Comment (Ron-Lavi) [reply package.json:23]: update all to `3.5.2` - the latest

**Comment by mmoll** [comment]
does this need an RPM update?

**Comment by jeremylenz** [comment]
@laviro Nice work on this, I knew I remembered seeing that error before :)

**Review by jeremylenz — DISMISSED** [review]
Tested and appears to fix the errors, thanks @laviro!

**Comment by ezr-ondrej** [comment]
[test foreman]

**Comment by sharvit** [comment]
> does this need an RPM update?

Yes

**Review by sharvit — DISMISSED** [review]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
I'm inclined to merge this to fix the travis tests, however the browser integration tests are still broken (although AFAICT in a different way as currently in develop). Could this get also fixed here?

File: package.json (hunk 1 above)
Comment (ekohl) [reply package.json:23]: https://github.com/theforeman/foreman-packaging/pull/4461

Edit: looks like I missed the other updates.

**Review by mmoll — APPROVED** [review]

**Comment by Ron-Lavi** [comment]
@mmoll @ekohl thanks for pushing this PR forward !

**Pull request merged**